docs/topics/assembly_deploying-the-booster-to-openshiftonline.adoc

```diff
@@ -7,6 +7,7 @@
 // Rationale: This procedure is identical in all deployments.
 
 [id='deploying-the-booster-to-openshiftonline_{context}']
+[id='deploying-the-example-to-openshiftonline_{context}']
 = Deploying the {parameter-mission-name} booster to {OpenShiftOnline}
```

Comment:
Deploying the {parameter-mission-name} booster --> Deploying the {parameter-mission-name} example application